packages/cli/src/index.ts

```diff
@@ -137,6 +137,13 @@ function attachCommand<IsDetached extends boolean>(
       const passedOptions = this.opts();
       const argv = Array.from(args).slice(0, -1);
 
+      // Copy parent commander verbose value to the sub-command if the sub-command also has a verbose option
+      // This is a workaround that can potentially be removed if commander is upgraded to v7+
```

Comment:
commander is now at v9. Is this fix relevant then? If so, please update the comment

Reply:
So I think we can use `enablePositionalOptions()`, this fixes this issue and we don't need to make workarounds.